src/scss/custom/components/_footer.scss

```diff
@@ -0,0 +1,164 @@
+footer {
```

Comment:
Ah I see. I have the tendency to use `<footer>` once and only as the page footer that's why my assumption was there would only be one `<footer>` in the DOM.

Should we just name this `page-footer`?